crates/update/src/cli/uninstall.rs

```diff
@@ -0,0 +1,22 @@
+use spacetimedb_paths::SpacetimePaths;
+
+use super::ForceYes;
+
+/// Uninstall an installed SpacetimeDB version.
+#[derive(clap::Args)]
+pub(super) struct Uninstall {
+    version: semver::Version,
+    #[command(flatten)]
+    yes: ForceYes,
+}
+
+impl Uninstall {
+    pub(super) fn exec(self, paths: &SpacetimePaths) -> anyhow::Result<()> {
+        let Self { version, yes } = self;
+        if yes.confirm(format!("Uninstall v{version}?"))? {
+            let dir = paths.cli_bin_dir.version_dir(&version);
+            std::fs::remove_dir_all(dir)?;
+        }
```

Comment:
if we uninstall the current version, will that leave something hanging in an awkward way? Should we remove the `current` link in that case?